week3/1-Tuesday/homework/exercise7.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+
+<head>
+
+    <h1>For this example:</h1>
+        <body>
+            <p>I want you to Google image search [something] and place [a] result here:</p>
+    
+            <img src="https://blog.metrostorage.com/wp-content/uploads/2015/06/FlagDay.jpg" alt="flag" width=400px>
+
+            <p>Use the "width" attribute to make the image 400 pixels wide</p>
+        </body>
+
+
+</head>
```

Comment:
Remember, "Head" is for things that don't get shown on a webpage, like loading css files and setting the tab name.